agents/che-core-api-agent/src/test/resources/run_launcher_bats_tests.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
Thanks @benoitf. At least someone has been indulgent today :-) However I will give it a try and tell you how it goes